app/data/JsonConversions.scala

```diff
@@ -118,7 +118,10 @@ object JsonConversions {
       )
   }
 
-  implicit val flagsWrites = {flags: Flags => (__ \ "legallySensitive").writeNullable[Boolean]}
+  implicit val flagsWrites = {flags: Flags =>
+    (__ \ "legallySensitive").writeNullable[Boolean] and
```

Comment:
As we discussed, we should think about how we do json conversions in general when we add the missing composer fields to the atom model.